packages/cli/src/eventbus/__tests__/audit-event-relay.service.test.ts

```diff
@@ -80,4 +82,67 @@ describe('AuditorService', () => {
 			},
 		});
 	});
+
+	it('should log on `workflow-post-execute` for successful execution', () => {
+		const payload = mock<Event['workflow-post-execute']>({
+			executionId: 'some-id',
+			success: true,
+			userId: 'some-id',
+			workflowId: 'some-id',
+			isManual: true,
+			workflowName: 'some-name',
+			metadata: {},
+			runData: mock<IRun>({ data: { resultData: {} } }),
+		});
+
+		eventService.emit('workflow-post-execute', payload);
+
+		const { runData: _, ...rest } = payload;
+
+		expect(eventBus.sendWorkflowEvent).toHaveBeenCalledWith({
+			eventName: 'n8n.workflow.success',
+			payload: rest,
+		});
+	});
+
+	it('should handle `workflow-post-execute` event for unsuccessful execution', () => {
+		const runData = mock<IRun>({
+			data: {
+				resultData: {
+					lastNodeExecuted: 'some-node',
+					// @ts-expect-error Partial mock
+					error: {
+						node: mock<INode>({ type: 'some-type' }),
+						message: 'some-message',
+					},
+					errorMessage: 'some-message',
+				},
+			},
+		}) as unknown as IRun;
```

Comment:
Open to suggestions, typing this was taking too long.

Reply:
We really should have a mocking function that allows creating partial mocks. Before that this is fine